com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/assist/ext/ResponseCodeContentAssistExt.java

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+ * Copyright (c) 2017 ModelSolv, Inc. and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    ModelSolv, Inc. - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package com.reprezen.swagedit.core.assist.ext;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.google.common.base.Predicate;
+import com.google.common.base.Strings;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import com.reprezen.swagedit.core.Activator;
+import com.reprezen.swagedit.core.assist.Proposal;
+import com.reprezen.swagedit.core.model.AbstractNode;
+import com.reprezen.swagedit.core.schema.TypeDefinition;
+
+public class ResponseCodeContentAssistExt implements ContentAssistExt {
+
+    private static final JsonPointer pointer = JsonPointer.compile("/definitions/responses");
+
```

Comment:
It works for OAS3, but does not work for Swagger v2. I will create a separate issue for this.

Reply:
Can you confirm it is not working for you for v2? I have it working for me.

![screen shot 2017-10-04 at 16 15 19](https://user-images.githubusercontent.com/148045/31180579-4af439ee-a91f-11e7-9aa4-62bfb71b0afd.png)